pom.xml

```diff
@@ -13,6 +13,7 @@
     <version>${revision}${changelist}</version>
     <name>Folder-based Authorization Strategy</name>
     <description>Manage access to Folders</description>
+    <url>https://github.com/jenkinsci/folder-auth-plugin</url>
```

Comment:
Role Strategy plugin uses the Wiki page for the URL. Is it fine to use the link to the GitHub repo here?

Reply:
+1. We consider changing the documentation publishing logic to take this URL into account leter